components/IscnUploadedInfo.vue

```diff
@@ -146,6 +159,10 @@ export default class IscnUploadedInfo extends Vue {
     return `ipfs://${this.ipfsHash}`
   }
 
+   get transactionsURL(){
+     return BIG_DIPPER_TRANSACTIONS + this.iscnHash
```

Comment:
Prefer template string
```suggestion
     return `BIG_DIPPER_TRANSACTIONS${this.iscnHash}`
```